### ISSUE TITLE
augmented reality on webrtc

### ISSUE THREAD

**Issue by nrlchsn** [muaz-khan/WebRTC-Experiment#465]
is there any way we can put augmented reality to the webrtc ? :D
so the AR will show to other that connect to a room hehehehe

**Comment by AndyNormore** [comment]
Yes. I'm working on this concept very soon at SkyRooms.IO which is using this engine.

**Comment by RedouaeElalami** [comment]
The same here ? did you find anything guys 
Thanks

**Comment by joeSIX** [comment]
I too have been looking for resources and information on this subject. If you can't find anything and want to start a collab on this, let me know.

**Comment by AndyNormore** [comment]
You can try https://threejs.org/

As a javascript overlay

**Comment by RamyaAshika** [comment]
Ya, I did. @joeSIX  @AndyNormore
